src/index.html

```diff
@@ -215,15 +224,91 @@ <h1 class="h-header h-txt24 js-lang" data-key="section-doswiadczenie">Doświadcz
                             <p class="h-txt16 js-lang" data-key="doswiadczenie-profesjonalizm">Profesjonalizm oraz doskonałe wyniki techniczne napraw maszyn trakcyjnych, które przechodzą oczekiwania naszych klientów, pozwoliły nam na stworzenie listy referencji, z której jesteśmy dumni.</p>
                         </div>
                     </div>
+
                     <div class="l-col-clean col-md-5">
                         <div class="doswiadczenie__icon-div">
                             <img class="doswiadczenie__icon" src="img/doswiadczenie__icon.png" alt="doswiadczenie__icon">
-                            <p class="js-doswiadczenie__x-lat h-header h-txt9 h-white text-center"></p>
                         </div>
                     </div>
+
+                </div>
+            </div>
+
+            <div class="cert">
+                <div class="l-container-padding row">
+
+                    <div class="l-col-clean col-sm-10 col-md-7">
+                        <div class="section__header">
+                            <h1 class="h-header h-txt24 h-orange js-lang" data-key="section-cert">Gwarantowana jakość<br>wykonania</h1>
+                        </div>
+                        <div class="section__p l-last-element">
+                            <p class="h-txt16 js-lang" data-key="cert-potencjal">Potencjał i doświadczona kadra gwarantują wysoką jakość produktów i usług oferowanych przez firmę. Spółka posiada certyfikat ISO 9001:2009 wydany przez Urząd Dozoru Technicznego (UDT-CERT).</p>
+                        </div>
+                    </div>
+
+                </div>
+                
+                <div class="l-container-padding row">
+
+                    <div class="l-col-clean col-lg-6">
+                        <ul class="c-list">
+                            <div class="c-list__title">
+                                <p class=" js-lang" data-key="cert-certificates">Nasze świadectwa:</p>
+                            </div>
+                            <li class="c-list__item c-list__item--spaced row">
+                                <a class="c-list-item__inner h-txt24 h-orange l-col-clean col-6 col-sm-8 js-lang" data-key="cert-pkp">Uznany wykonawca usługi</a>
+                                <img class="c-list-item__inner-icon  c-list-item__inner-icon--pkp" src="img/cert__icon--pkp.png" alt="cert__icon--iso">
```

Comment:
Disclaimer: not the scope of this PR

To avoid visible pixels please use 2 times larger images.
![image](https://user-images.githubusercontent.com/789841/45809515-a2772b00-bcc8-11e8-8151-66a5ebd4762e.png)


Example of much larger image on different website:
![image](https://user-images.githubusercontent.com/789841/45809492-98552c80-bcc8-11e8-8155-a2b228cafae1.png)

Reply:
I think all the new images/icons are 2 times larger... Possibly there are some leftovers from first commits, and they will be replaced in the `feature/generalCleanup` which is the next PR comming ;)